composer.json

```diff
@@ -25,9 +25,9 @@
         "symfony/phpunit-bridge": "^3.4 || ^4.0",
         "doctrine/orm": "^2.6",
         "symfony/monolog-bundle": "^3.1",
-        "doctrine/doctrine-fixtures-bundle": "^3.0",
+        "doctrine/doctrine-fixtures-bundle": "^3.0 >=3.0.2",
```

Comment:
This line shouldn't be in `require` section instead of `require-dev` ? 
Since we use it in `Services\FixturesLoaderFactory`, it's became a dependency, WDYT ?

Got this error when updating to 2.0.0-alpha5

```
The service "liip_functional_test.services.fixtures_loader_factory" has a dependency on a non-existent service "doctrine.fixtures.loader". 
```

Reply:
see https://github.com/liip/LiipFunctionalTestBundle/pull/437